src/utilities/utilities.ts

```diff
@@ -267,15 +272,17 @@ export function swiftDriverTargetFlags(indirect = false): string[] {
 
     const sdkKindParts = configuration.sdk.split("/");
     const sdkKind = sdkKindParts[sdkKindParts.length - 1];
-    if (sdkKind.includes(IPHONE_SDK_KIND)) {
-        // Obtain the iOS version of the SDK.
-        const version = sdkKind.substring(
-            // Trim the prefix
-            sdkKind.length - IPHONE_SDK_KIND.length,
-            // Trim the `.sdk` suffix
-            sdkKind.length - 4
-        );
-        args = ["-target", `arm64-apple-ios${version}`];
+    for (const target of targetMap) {
+        if (sdkKind.includes(target.name)) {
+            // Obtain the iOS version of the SDK.
```

Comment:
```suggestion
            // Obtain the version of the SDK.
```
This is no longer just iOS